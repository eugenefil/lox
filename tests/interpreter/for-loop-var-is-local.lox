for ch in "bar" {}
ch;
