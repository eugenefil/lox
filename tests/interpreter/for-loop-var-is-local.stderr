error: identifier 'ch' is not defined
 --> $DIR/for-loop-var-is-local.lox:2:1
  |
2 | ch;
  | ^^
